docs/commands/lint.md

```diff
@@ -173,12 +214,11 @@ redocly lint --format=checkstyle
 </checkstyle>
 ```
 
-In this format, `lint` uses the [Checkstyle](https://checkstyle.org/) XML report format.
 Due to the limitations of this format, only file name, line, column, severity,
 and rule ID (in the `source` attribute) are included. All other information is
 omitted.
 
-### Max problems
+### Limit the problem count
```

Comment:
If you change this, change the corresponding deep anchor link on line 34.

Reply:
Ahhh, there's a few of these! Thanks!